server/routes/courtRegister/courtMapper.ts

```diff
@@ -98,6 +97,6 @@ export function courtsPageMapper(courtsPage: CourtsPage): CourtsPageView {
 }
 
 export type AllCourtsFilter = {
-  courtTypeIds: string[]
+  courtTypeIds?: string[]
```

Comment:
so would this be null rather than an empty list?

Reply:
That's just the type - doesn't initialise or anything.

Reply:
But yes, it was implemented as nullable on the API and doesn't check for an empty list) so needs to be the same here.